beamer/chain.py

```diff
@@ -216,7 +216,7 @@ def process_requests(context: Context) -> None:
         elif request.is_filled:
             claim_request(request, context)
 
-        elif request.is_withdrawn:
+        elif request.is_withdrawn or request.is_ignored:
             active_claims = any(claim.request_id == request.id for claim in context.claims)
             if not active_claims:
                 log.debug("Removing withdrawn request", request=request)
```

Comment:
Nitpick: we should consider updating the log message now that also ignored requests are removed.